dup_algorithm.rs

```diff
@@ -0,0 +1,298 @@
+#![allow(unknown_lints)]
+#![warn(clippy::all)]
+
+extern crate ndarray;
```

Comment:
Interesting -- how so the duplication?

Reply:
see above. It was code for local testing. will remove it again